noxfile.py

```diff
@@ -52,10 +52,19 @@ def install(session: nox.Session, group: str) -> None:
         session.run(*"git clone https://github.com/laminlabs/lamin-cli".split())
         session.run(*"pip install lamin-cli".split())
     elif group == "hub-local":
-        session.run(*"pip install -e .[aws,dev,hub]".split())
-        session.run(*"pip install ./laminhub/rest-hub".split())
+        session.run(*"pip install lamindb".split())
+        if os.getenv("GITHUB_ACTIONS") is None:
```

Comment:
Also this looks way too complicated, there gotta be a simpler solution for this.